models/h2o_dai_german_credit/README.md

```diff
@@ -23,7 +23,7 @@ Specifically, it shows how to use the Certifai toolkit to do the following for t
 ## Usage Instructions
 
 ### Pre-requisites
-- Scoring Pipeline ([Mojo](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/scoring-pipeline-cpp.html#examples) or [Python](https://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/scoring-standalone-python.html#quick-start)) downloaded to disk on a Ubuntu 16.04 machine with Python 3.6. Refer to 
+- Scoring Pipeline ([Mojo](http://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/scoring-pipeline-cpp.html#examples) or [Python](https://docs.h2o.ai/driverless-ai/latest-stable/docs/userguide/scoring-standalone-python.html#quick-start)) downloaded to disk on a Ubuntu 16.04 machine with Python 3.6. Refer to the docs linked above to create and setup the environment to run this scoring pipeline
 
 ### Environment Setup
 
```

Comment:
In the Environment Setup section and through the instructions, I suggest:
a) Tell reader what specific environment names we are going to use in the instructions, e.g. `certifai` for the scanner and `daimodel` for the prediction service.  Also tell them what each environment is for, i.e.  one for running scanner, one for running the prediction service.
b) Include the appropriate conda activate commands in the instructions